### PR TITLE
cli: PID minter option

### PR DESCRIPTION
**Pull request by jirikuncar**
* NEW Adds option to specify PID minters during record creation if Invenio-PIDStore package is installed.

--
- cc @tiborsimko @lnielsen @switowski @hachreak

### CODE REVIEW

**Comment by tiborsimko** [comment]
OT1H I like the minter option, which addresses #107 motivated concerns too.

OTOH, an Invenio installation might be in a situation where it receives "forced" control numbers from the master system, coming via incoming MARC files, so I'm also OK for having 001 in the incoming file kind of situation.  This may come handly for large-scale demo site integration tests, since we know in advance which record will receive which PID.

**Comment by jirikuncar** [comment]
@tiborsimko I'm keeping only the `cli` commit. But I still think that for our demo data we should avoid fixed control numbers as it might be easier for later re-use.

**Comment by jirikuncar** [comment]
@lnielsen If https://github.com/inveniosoftware/invenio-pidstore/pull/59 gets merged first we might need to release Invenio-PIDStore before.

**Comment by lnielsen** [comment]
PIDStore release coming - inveniosoftware/invenio-pidstore#61

File: setup.py [hunk 1]
```diff
@@ -50,6 +50,9 @@
     'access': [
```
Comment (lnielsen) [inline setup.py]: Copyright year

File: examples/app.py [hunk 2]
```diff
@@ -55,6 +55,7 @@
 
 import os
 
+import pkg_resources
```
Comment (lnielsen) [inline examples/app.py]: Copyright year

**Comment by lnielsen** [comment]
Sorry, didn't spot the copyright years in the preview review

**Comment by jirikuncar** [comment]
@lnielsen hups. I had them fixed but haven't pushed. There are more little things fixed ...

**Pull request merged**